armory/paths.py

```diff
@@ -35,10 +35,9 @@
 
 DATASETS = config.get("datasets") or os.path.join(USER_ARMORY, "datasets")
 EXTERNAL_REPOS = os.path.join(DATASETS, "external_repos")
-MODELS = config.get("models") or os.path.join(USER_ARMORY, "models")
 TMP = config.get("tmp") or os.path.join(USER_ARMORY, "tmp")
 OUTPUTS = config.get("outputs") or os.path.join(USER_ARMORY, "outputs")
 
-DOCKER = "/root"
+DOCKER = "/"
```

Comment:
This breaks pulling from datasets in the container --> those go to `/root/.armory`, since armory.data.datasets uses paths.DATASETS, which uses USER_ARMORY as the base (this is the local user outside the container and "root" in the container).

Does writing to "/root" do anything to the host outside of the mounted directories?

Reply:
Updated code so that we no longer use any "user" paths within the container. All docker paths as explicit.

Since we're running the container as host user there is no write permission to anything under `/root` inside of the container. I've merged this PR with master so make sure you `sudo rm -R ~/.armory` on host prior running for the first time.